karate-core/src/main/java/com/intuit/karate/core/ScriptBridge.java

```diff
@@ -596,6 +596,30 @@ public ScriptValue fromObject(Object o) {
         return context.getScenarioInfo();
     }
 
+    public Map<String, Object> getScenario() {
```

Comment:
thanks everything else looks good. just this one question.

instead of returning a `Map` can we return the `Scenario` instance itself. as far as I know, the getters will continue to work, e.g. `karate.scenario.exampleIndex`

Reply:
yep we can do that.  i didn't do that initially as i wasn't sure if you'd want to expose the whole object.  also for me returning the Scenario object i had to call it as shown below in karate-config.js.

karate-config.js
karate.log(karate.getScenario().getExampleIndex());

output running scenario outline
{"timestamp":"2020-08-11 08:26:49.293","level":"INFO","thread":"ForkJoinPool-1-worker-3","logger":"com.intuit.karate","message":"0 ","context":"default"}
{"timestamp":"2020-08-11 08:26:49.293","level":"INFO","thread":"ForkJoinPool-1-worker-5","logger":"com.intuit.karate","message":"1 ","context":"default"}
{"timestamp":"2020-08-11 08:26:49.293","level":"INFO","thread":"ForkJoinPool-1-worker-7","logger":"com.intuit.karate","message":"2 ","context":"default"}

output running scenario
{"timestamp":"2020-08-11 08:27:42.709","level":"INFO","thread":"ForkJoinPool-1-worker-3","logger":"com.intuit.karate","message":"-1 ","context":"default"}